### ISSUE TITLE
Can't configure InstantMatchEnabled when creating a link token.

### ISSUE THREAD

**Issue by francisco-dechoudens** [viceroypenguin/Going.Plaid#71]
When trying to create a LinkToken (LinkTokenCreateAsync) with InstantMatchEnabled = true the following error appears:

"auth type select should only be enabled with the auth_type_select_enabled field, please remove the flow_type field"

```
var response = await _plaidClient.LinkTokenCreateAsync(
                    new LinkTokenCreateRequest()
                    {
                        User = new LinkTokenCreateRequestUser { ClientUserId = userId },
                        ClientName = "Hello World!",
                        Products = _configuration!.Products!.Split(',').Select(p => Enum.Parse<Products>(p, true)).ToArray(),
                        Language = Language.English,
                        CountryCodes = _configuration!.CountryCodes!.Split(',').Select(p => Enum.Parse<CountryCode>(p, true)).ToArray(),
                        Auth = new LinkTokenCreateRequestAuth {
                            InstantMatchEnabled = true
                        }
                    });

```

**Comment by viceroypenguin** [comment]
The FlowType property was not marked as nullable - slight disagreement between the `string` representation in the yaml from plaid and the `enum` interpretation generated by my code. I've updated to v4.11.1, which will now ignore the FlowType property. This should address your issue.

**Comment by jjgillespie** [comment]
This issue has resurfaced in v4.13.0 which generated new code for the "Update to Plaid v1.161.5". The FlowType property is once again not nullable on LinkTokenCreateRequestAuth.  Downgrading to v4.12.0 resolves the issue for now.

**Comment by viceroypenguin** [comment]
Shoot, sorry. Wasn't paying attention with the merge. There's a new open PR for the plaid API, so I'll get it fixed with that one in v4.14.0. Again, sorry!

**Comment by viceroypenguin** [comment]
@jjgillespie v4.14.0 is released with the fix restored.
